app-modules/form/src/Filament/Resources/FormResource/Pages/ManageFormWorkflows.php

```diff
@@ -0,0 +1,118 @@
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Form\Filament\Resources\FormResource\Pages;
+
+use AdvisingApp\Form\Filament\Resources\FormResource;
+use AdvisingApp\Form\Models\Form;
+use AdvisingApp\Workflow\Filament\Resources\WorkflowResource;
+use AdvisingApp\Workflow\Models\Workflow;
+use AdvisingApp\Workflow\Models\WorkflowTrigger;
+use App\Features\WorkflowFeature;
+use App\Models\User;
+use Filament\Actions\Action;
+use Filament\Resources\Pages\ManageRelatedRecords;
+use Filament\Tables\Actions\EditAction;
+use Filament\Tables\Columns\IconColumn;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+
+class ManageFormWorkflows extends ManageRelatedRecords
+{
+    protected static string $resource = FormResource::class;
+
+    protected static string $relationship = 'workflows';
+
+    public static function canAccess(array $arguments = []): bool
+    {
+        return WorkflowFeature::active();
+    }
+
+    public static function getNavigationLabel(): string
+    {
+        return 'Workflows';
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->recordTitleAttribute('id')
+            ->columns([
+                TextColumn::make('name'),
+                IconColumn::make('is_enabled')
+                    ->label('Enabled')
+                    ->state(fn (Workflow $record) => $record->is_enabled)
+                    ->boolean(),
+            ])
+            ->actions([
+                EditAction::make(),
+            ])
+            ->recordUrl(fn (Workflow $record) => WorkflowResource::getUrl('edit', [$record]));
+    }
+
+    /**
+     * @return array<Action>
+     */
+    public function getHeaderActions(): array
+    {
+        return [
+            $this->createWorkflowAction(),
+        ];
+    }
+
+    public function createWorkflowAction(): Action
+    {
+        return Action::make('create')
+            ->label('Create New Workflow')
+            ->action(function () {
+                $workflowTrigger = WorkflowTrigger::create([
+                    'type' => 'Time Based',
+                    'related_type' => Form::class,
+                    'related_id' => $this->getOwnerRecord()->getKey(),
+                    'created_by_type' => User::class,
+                    'created_by_id' => auth()->user()->getKey(),
+                ]);
```

Comment:
Turn the `WorkflowTrigger::create` into `WorkflowTrigger::make` then set the related and created by relationships using the relationship methods then save the workflow